transcendence-app/src/chat/chat.domain.ts

```diff
@@ -0,0 +1,19 @@
+/* eslint-disable @typescript-eslint/no-inferrable-types */
+import { Exclude } from 'class-transformer';
+import { ChatEntity } from './infrastructure/db/chat.entity';
+
+export class Chat {
```

Comment:
Will Chat and ChatRoom have the same domain model? we could export in the future , both models, Chat and ChatRoom, so I'd rename this to `ChatRoom`